src/components/EditCoursePage/index.jsx

```diff
@@ -138,6 +119,17 @@ class EditCoursePage extends React.Component {
       courseRunOptions,
     } = this.props;
     const { startedFetching } = this.state;
+    const minimalCourseRuns = course_runs && course_runs.map(courseRun => ({
```

Comment:
I think you need staff in the fields here? Hopefully that will fix the issue I'm seeing where staff aren't loaded even when they're returned from discovery

Reply:
👍